templates/docs/examples/patterns/side-navigation/_icons.html

```diff
@@ -1,95 +1,112 @@
-    </li>
-    <li class="p-side-navigation__item ">
-      <a class="p-side-navigation__link" href="#"><span class="u-truncate">First level link with label that is truncated because it's long long long long long long long</span><div class="p-side-navigation__status"><span class="p-label--validated">Validated</span></div></a>
-    </li>
-  </ul>
-  <ul class="p-side-navigation__list">
-    <li class="p-side-navigation__item--title">
-      <span class="p-side-navigation__text">Title that is not a link</span>
-    </li>
-    <li class="p-side-navigation__item">
-      <span class="p-side-navigation__text">First level text</span>
-    </li>
-    <li class="p-side-navigation__item">
-      <span class="p-side-navigation__text">Text item with children</span>
-      <ul class="p-side-navigation__list">
-        <li class="p-side-navigation__item">
-          <a class="p-side-navigation__link" href="#">Second level link</a>
-        </li>
-        <li class="p-side-navigation__item ">
-          <span class="p-side-navigation__text">Second level text</span>
-        </li>
-        <li class="p-side-navigation__item ">
-          <span class="p-side-navigation__text">Second level text with children</span>
-          <ul class="p-side-navigation__list">
-            <li class="p-side-navigation__item">
-              <span class="p-side-navigation__text">Third level text</span>
-            </li>
-            <li class="p-side-navigation__item ">
-              <a class="p-side-navigation__link is-active" href="#">Third level active item that is long and wraps wraps wraps wraps wraps wraps</a>
-            </li>
-            <li class="p-side-navigation__item ">
-              <a class="p-side-navigation__link" href="#"><span class="u-truncate">Third level link that is truncated because it's long long long long long long long</span></a>
-            </li>
-            <li class="p-side-navigation__item ">
-              <a class="p-side-navigation__link" href="#">Third level link</a>
-            </li>
-          </ul>
-        </li>
-      </ul>
-    </li>
-    <li class="p-side-navigation__item ">
-      <span class="p-side-navigation__text">First level item that is not a link</span>
-    </li>
-    <li class="p-side-navigation__item ">
-      <a class="p-side-navigation__link" href="#">First level link</a>
-    </li>
-  </ul>
+<div class="p-side-navigation--icons {% if is_dark %}is-dark{% endif %}" id="drawer-icons">
+  <button class="p-side-navigation__toggle js-drawer-toggle" aria-controls="drawer-icons">
+    Toggle side navigation
+  </button>
+
+  <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="drawer-icons"></div>
+
+  {% if is_dark %}
+  <!-- dark background color should be changed via color theme variables, inline style is used here for example purposes -->
+  {% endif %}
```

Comment:
Do we need this?

Reply:
I wanted to make it clear that setting background inline is not recommended way. Just in case someone uses example as a reference.

I can remove it. Especially that this example is not exposed in CodePen embeds, it's just listed on Examples pages and screenshoted for Percy.